### DISCUSSION TITLE
post/kakaopay-techlog-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#44] [Announcements]
# post/kakaopay-techlog-2/

카카오페이 기술 블로그(Tech Log)는 어떻게 운영될까요? Tech Log만의 운영 노하우와 경험을 공유합니다.

https://tech.kakaopay.com/post/kakaopay-techlog-2/

**Comment by yozmngooddev** [comment]
테크블로그를 고민하거나 운영하신 분들에게 많은 도움이 될 것 같아요.
감사합니다.

**Comment by hg-pyun** [comment]
배워갑니다.
